### DISCUSSION TITLE
How to deploy a virtualized Geonode using VirtualBox?

### DISCUSSION THREAD

**Discussion by lcoandrade** [GeoNode/geonode#11415] [Q&A]
I need to deploy an Geonode installation at my University. They work with a virtualized environment. I already have Geonode working at localhost in my VirtualBox. What should I do to make this Geonode available through an address like: `https://www.myunivesity.edu/geonode`? Is there some tutorial to follow to achieve that?

Thanks in advance!

**Comment by afabiani** [comment]
Take a look at the variable `FORCE_SCRIPT_NAME` you should set it to `geonode` other than updating your `NGINX` configuration

**Reply by ridoo** [reply]
Interesting. Did not stumble over this one in the settings/documentation. Is there some background/history why naming it this way?

**Comment by lcoandrade** [comment]
Where do I find this variable?

**Reply by t-book** [reply]
voila

https://github.com/GeoNode/geonode/blob/ec457bcd4b04247a4639857d829be91c2b99e6b6/geonode/settings.py#L68
